### ISSUE TITLE
Docs uploads are failing

### ISSUE THREAD

**Issue by ocefpaf** [ODM2/ODM2PythonAPI#101]
@asmeurer and @gforsyth if you have some time can you take a quick look at my config here?
I guess I may be doing something stupid, but I was not able to configure `doctr` properly for this project. I am using 1.6.1 btw.

```
build succeeded.
Link check complete; look for any errors in the above output or in build/linkcheck/output.txt.
~/build/ODM2/ODM2PythonAPI
Setting git attributes
git config --global user.name 'Doctr (Travis CI)'
git config --global user.email drdoctr@users.noreply.github.com
Adding doctr remote
ERROR: The doctr command '/home/travis/miniconda/envs/TEST/lib/python3.6/site-packages/doctr/__main__.py deploy --sync .' failed: DOCTR_DEPLOY_ENCRYPTION_KEY environment variable is not set
git checkout 627d74959f2632603667ea9e939236e8fd750085
HEAD is now at 627d749... Merge pull request #99 from ocefpaf/docs
git stash pop
No stash found.
Error: DOCTR_DEPLOY_ENCRYPTION_KEY environment variable is not set
```

**Comment by asmeurer** [comment]
Did this just start failing after the doctr release?

**Comment by asmeurer** [comment]
In the past, that error sometimes is intermittent and goes away after rebuilding. If not, you can usually fix it by regenerating the deploy key with `doctr configure`.

**Comment by ocefpaf** [comment]
> Did this just start failing after the doctr release?

No, this is a new deployment that did not use `doctr` before. (I followed the same workflow as the oher deployments I made with the previous version though.)

I changed the the position of the \` the first time, b/c if I copy-n-paste `doctr`'s screen output it fails

`doctr` suggests:

```yaml
env:
  global:`
    - secure:
```

I had
```yaml
env:
  global:
    - secure:`
```

See https://github.com/ODM2/ODM2PythonAPI/compare/bc9a2589ee39...fa8081e154bc

Also, the new encrypted var is quite long, so it is easy to have issue when copying, line line lines or even not selecting some parts. It would be nice to have an option to write that directly to Travis-CI.
